packages/dbml-parse/tests/lexer/input/unclosed_strings.in.dbml

```diff
@@ -4,6 +4,4 @@
 
 '''This is also an unclosed string
 
-\''''''
```

Comment:
Please help add at least one more test for case of `'''... \''''` to show that it work